<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-7KBB9M5" failures="1" tests="1" name="Test" time="34.823" errors="0" timestamp="03 Aug 2022 11:34:44 GMT">
  <testcase classname="com.swaglabs.tests.SauceLabCheckoutCompletePageTests" name="validateOrderSuccessMessage" time="28.779">
    <failure type="java.lang.AssertionError" message="
expected: &amp;quot;THANK YOU FOR YOUR ORDE&amp;quot;
 but was: &amp;quot;THANK YOU FOR YOUR ORDER&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
expected: "THANK YOU FOR YOUR ORDE"
 but was: "THANK YOU FOR YOUR ORDER"
at com.swaglabs.tests.SauceLabCheckoutCompletePageTests.validateOrderSuccessMessage(SauceLabCheckoutCompletePageTests.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- validateOrderSuccessMessage -->
  <testcase classname="com.swaglabs.tests.BaseTest" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.swaglabs.tests.BaseTest" name="@BeforeSuite beforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
</testsuite> <!-- Test -->
